hypothesis/checkmate
# Add the requirements and local dev services for Postgres and Rabbit

For: https://github.com/hypothesis/checkmate/issues/37

This PR is just to clear the way to start celery and postgres work locally, and to cut down on the waffle in following PRs.

This includes the stuff required to get postgres and rabbit working locally, along with some pretty likely requirements for using them.

There was also a tweak to `touch` the requirements files before compiling them. This gets around a nasty situation where we can't create the files because the env to create them won't start because they aren't there...

To test:

 * `make requirements`  - Shouldn't do much
 * `make services` - Should start Postgres and Rabbit
